IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormCommitTests.cs

```diff
@@ -214,6 +214,9 @@ public void Should_handle_well_commit_message_in_commit_message_menu()
             _referenceRepository.CreateCommit("Too long commit message that should be shorten because first line of a commit message is only 50 chars long");
             RunFormTest(form =>
             {
+                // Await updated FileViewer
+                ThreadHelper.JoinPendingOperations();
```

Comment:
I would rather add this to the method `RunFormTest` or even to `UITest.RunForm`.

Reply:
Trying UITest